### DISCUSSION TITLE
Unable to create a payout for the same BTC address that was used before

### DISCUSSION THREAD

**Discussion by ilearnio** [btcpayserver/btcpayserver#4844] [Ideas / Feature Requests]
I am trying to create a payout using Greenfield API, which works well, however I am unable to create a second payout for the same BTC address twice because getting an error "This address is already used for another payout". Why is it a problem to initiating withdrawals for the same address? Maybe there is an option disable that check?

**Comment by Kukks** [comment]
This only applies when the other payout is still active. You may reuse the address when the other payout is in cancelled or completed state.

The reason behind this is that if you have two payouts to the same address, how would you know which payment was meant for which? 

In any case reusing addresses is a bad practice that should be avoided.

**Comment by ilearnio** [comment]
> if you have two payouts to the same address, how would you know which payment was meant for which?

@Kukks I am creating a payout with a name like "Withdrawal: {ID}", and the ID of the withdrawal is registered in our system and has whole info. So what we are trying to do is to allow our customers to withdraw a specific amount from their account balance, so user may want to do multiple withdrawals to the same address and it would be strange if we will require him to enter a new BTC address every time.

**Reply by wesr2424** [reply]
how do i do a pay out to an address from my paylink

**Reply by Kukks** [reply]
Users should generate a new address every time. I understand it is inconvenient for those that do not understand the implications but it is still a bad practice to reuse. You may store additional data around the payout, but the payment data itself could conflict.

Anyways, we're quite certain about not changing this mechanic. From your end, you can always cancel the withdrawal and create a new payout with the combined amount.
